src/getHumanAge.test.js

```diff
@@ -4,7 +4,26 @@ describe('getHumanAge', () => {
   const { getHumanAge } = require('./getHumanAge');
 
   test('should be declared', () => {
-    expect(getHumanAge)
-      .toBeInstanceOf(Function);
+    expect(getHumanAge).toBeInstanceOf(Function);
+  });
+
+  test('should return [0, 0] when catAge and dogAge are 0', () => {
+    expect(getHumanAge(0, 0)).toEqual([0, 0]);
+  });
+
+  test('should return [0, 0] when catAge and dogAge are less than 15', () => {
+    expect(getHumanAge(14, 14)).toEqual([0, 0]);
+  });
```

Comment:
what about cases, when we have 15,18,23,24 years?) Please, it would be good to add this checks